client/src/views/Home.vue

```diff
@@ -136,14 +129,20 @@ export default {
     uploaded(uploads) {
       this.uploaderDialog = false;
 
-      // transcode video
-      const transcodes = uploads.filter(({ results }) => {
-        const videos = results.filter((result) => ['avi', 'mp4', 'mov', 'mpg'].includes(result.exts[0]));
-        videos.forEach((result) => {
-          this.girderRest.post(`/viame/conversion?itemId=${result.itemId}`);
-        });
-        return !!videos.length;
+      // Check if any transcoding should be done
+      const transcodes = uploads.filter(({ results, folder }) => {
+        const videoTranscodes = results
+          .filter(({ name }) => videoFilesRegEx.test(name))
+          .map(({ itemId }) => runVideoConversion(itemId));
+        const imageTranscodes = results
+          .filter(({ name }) => !webFriendlyImageRegEx.test(name) && imageFilesRegEx.test(name));
+
+        if (imageTranscodes) {
+          runImageConversion(folder._id);
+        }
```

Comment:
Do you want the length of `imageTranscodes` not the actual result because wouldn't that be an empty array?  This is causing an image transcode task to be run on the EHU data.

Reply:
Thanks, fixed.